indigo_app/templates/indigo_api/task_list.html

```diff
@@ -26,7 +26,7 @@
     {% if tasks %}
 
       {% if form.format.value == 'list' %}
-        {% include 'indigo_api/_task_list.html' with tasks=tasks %}
+        {% include 'indigo_api/_task_list_all.html' with tasks=tasks selectable=True %}
```

Comment:
Let's replace `_task_list.html` with the new code above, no need for a second template.

Reply:
Done